architecture/schema.sql

```diff
@@ -9,7 +9,7 @@ DROP TYPE IF EXISTS magnitude_type;
 
 CREATE TYPE alert_level AS ENUM ('green', 'yellow', 'orange', 'red');
 CREATE TYPE location_source AS ENUM ('ak', 'at', 'ci', 'hv', 'ld', 'mb', 'nc', 'nm', 'nn', 'pr', 'pt', 'se', 'us', 'uu', 'uw');
-CREATE TYPE magnitude_type AS ENUM ('Md', 'Ml', 'Ms', 'Mw', 'Me', 'Mi', 'Mb', 'MLg');
+CREATE TYPE magnitude_type AS ENUM ('Md', 'Ml', 'Ms', 'Mw', 'Me', 'Mi', 'Mb', 'MLg', 'Mww');
```

Comment:
```suggestion
CREATE TYPE magnitude_type AS ENUM ('Md', 'Ml', 'Ms', 'Mw', 'Me', 'Mi', 'Mb', 'MLg', 'Mww', 'Mwc', 'Mwb', 'Mwr', 'Mfa', 'Mh');
```
It could be more flexible to add these types, based on the USGS's Magnitude Type page, if we've missed some in the past?
[Possibly also 'Finite Fault' and 'Mint'?](https://www.usgs.gov/programs/earthquake-hazards/magnitude-types) although I'm not sure if they're in the data or just rarer than the rest.

Reply:
All the Mw's might come under the Mw but Mh might still be necessary?